### ISSUE TITLE
ENOENT error reported

### ISSUE THREAD

**Issue by cmoulliard** [csantanapr/grunt-cordovacli#28]
Hi,

I can run cordova client deployed as a node module under my project 

```
dabou:~/MyProjects/feedhenry-helloworld/cordova-client$ /Users/chmoulli/MyProjects/feedhenry-helloworld/cordova-client/node_modules/cordova/bin/cordova build browser
Running command: /Users/chmoulli/MyProjects/feedhenry-helloworld/cordova-client/platforms/browser/cordova/build 
Cleaning Browser project
Browser packaged app built in platforms/browser/build/package.zip
```

but not using grunt task

```
grunt cordovacli:build_browser
Running "cordovacli:build_browser" (cordovacli) task
Using cordova CLI version (5.3.3) 
Setting Current Working Directory (CWD) to hellocordova
Running:/Users/chmoulli/MyProjects/feedhenry-helloworld/cordova-client/node_modules/cordova/bin/cordova build browser

Error: spawn /Users/chmoulli/MyProjects/feedhenry-helloworld/cordova-client/node_modules/cordova/bin/cordova ENOENT
    at exports._errnoException (util.js:874:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:178:32)
    at onErrorNT (internal/child_process.js:344:16)
    at doNTCallback2 (node.js:439:9)
    at process._tickCallback (node.js:353:17)
    at Function.Module.runMain (module.js:469:11)
    at startup (node.js:134:18)
    at node.js:961:3



Execution Time (2015-10-14 04:41:53 UTC)
loading tasks              3ms  ▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇▇ 18%
cordovacli:build_browser  13ms 
Total 17ms
```

**Comment by cmoulliard** [comment]
If I run a spawn process locally, then that works too

file spawn.js

````
var spawn = require('child_process').spawn,
    cordova = spawn('cordova', ['build', 'browser']);

cordova.stdout.on('data', function (data) {
    console.log('stdout: ' + data);
});

cordova.stderr.on('data', function (data) {
    console.log('stderr: ' + data);
});

cordova.on('close', function (code) {
    console.log('child process exited with code ' + code);
});
````

and 

````
node spawn.js 
stdout: Running command: /Users/chmoulli/MyProjects/feedhenry-helloworld/cordova-client/platforms/browser/cordova/build 

stdout: Cleaning Browser project

stdout: Browser packaged app built in platforms/browser/build/package.zip

child process exited with code 0

````

**Comment by csantanapr** [comment]
I was able to reproduce with 5.3.3, but it's fix in cordova 3.4.0+ please update cordova cli
